rhel_aws/README.md

```diff
@@ -1,19 +1,11 @@
 # RHEL 8 Workshop - AWS Version
 
-![ansible](img/Ansible-Tower-Logotype-Large-RGB-FullGrey-300x124.png)
+![rhel 8](img/Logo-Red_Hat-Enterprise_Linux_8-B-Standard-RGB.png)
 
 `rhel8_aws` is a playbook to provision RHEL 8 in AWS. This playbook uses Ansible to provision AWS infrastructure and nodes.
 
 These modules all require that you have AWS API keys available to use to provision AWS resources. You also need to have IAM permissions set to allow you to create resources within AWS. There are several methods for setting up your AWS environment, on you local machine.
 
```

Comment:
These modules all require that you have AWS API keys to provision AWS resources.
Set IAM permissions to enable creation of resources within AWS.
Remove the comma after environment
the last "you" should be "your"